lib/Lex/PPMacroExpansion.cpp

```diff
@@ -1268,7 +1268,7 @@ static bool HasFeature(const Preprocessor &PP, StringRef Feature) {
       .Case("underlying_type", LangOpts.CPlusPlus)
       .Case("capabilities", PP.getTargetInfo().SupportsCapabilities())
       .Case("pointer_interpretation", PP.getTargetInfo().SupportsCapabilities())
-      .Case("__cheri_cast", PP.getTargetInfo().SupportsCapabilities())
+      .Case("__cheri_ptr", PP.getTargetInfo().SupportsCapabilities())
```

Comment:
The __has_feature check in CheriBSD will also need updating. Although we can probably just delete it now, I don't think CheriBSD compiles with any compiler version that doesn't include __cheri_cast